weblab_cg/_generator_chaste_cg.py+

```diff
@@ -20,12 +20,13 @@ class ChasteModel(object):
     # TODO: script to call generator from command line
     # TODO: Model types Opt, Analytic_j, Numerical_j, BE
 
-    _MEMBRANE_VOLTAGE_INDEX = 0
-    _CYTOSOLIC_CALCIUM_CONCENTRATION_INDEX = 1
-    _OXMETA = 'https://chaste.comlab.ox.ac.uk/cellml/ns/oxford-metadata#'
-    _PYCMLMETA = 'https://chaste.comlab.ox.ac.uk/cellml/ns/pycml#'
+    _MEMBRANE_VOLTAGE_INDEX = 0  # default index for voltage in state vector
+    _CYTOSOLIC_CALCIUM_CONCENTRATION_INDEX = 1  # default index for voltage in state vector
```

Comment:
This one isn't voltage :)

Reply:
Thanks ... a bit too trigger happy with the copy-paste